### PR TITLE
Add project conf

### PR DESCRIPTION
**Pull request by kjlundsgaard**
Allows Freight plugin to render in release-tracking page when we convert page to react in this PR https://github.com/getsentry/sentry/pull/5403#discussion_r117363669

### CODE REVIEW

File: src/sentry_freight/plugin.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ class FreightPlugin(ReleaseTrackingPlugin):
     description = 'Integrate Freight release tracking.'
     version = sentry_freight.VERSION
 
+    def has_project_conf(self):
```
Comment (macqueen) [inline src/sentry_freight/plugin.py:56]: is this going to cause the plugin to have that `Configure plugin` link on the `All integrations` page?

Comment (kjlundsgaard) [reply src/sentry_freight/plugin.py:56]: Yes, but when I remove it, the Freight plugin doesn't show in the Release Tracking view. Let me see if I can edit the PluginList component to get this to render without that.

Comment (kjlundsgaard) [reply src/sentry_freight/plugin.py:56]: I updated the plugins endpoint to not require these updates to show freight in the release tracking view: https://github.com/getsentry/sentry/pull/5403/files#diff-afd6044965ad3dff0908c1c857909b77R16

Comment (macqueen) [reply src/sentry_freight/plugin.py:56]: 👍

**Pull request closed without merging**